api/api.go

```diff
@@ -572,6 +574,11 @@ type CloudBackupGroupSchedCreateRequest struct {
 	MaxBackups uint
 	// Full indicates if scheduled backups must be full always
 	Full bool
+	// RetentionDays is the number of days that the scheduled backups will be kept
+	// and after these number of days it will be deleted
+	RetentionDays uint32
+	// Update indicates if existing schedule is being updated
+	Update bool
```

Comment:
We should create new APIs for update instead of overloading the Create APIs

Reply:
OK. Added new API for update.